lib/src/test/kotlin/network/account/AccountTest.kt

```diff
@@ -393,9 +393,9 @@ class AccountTest {
     }
 
     @Test
-    fun `get ETH balance`() {
+    fun getETHBalance() {
```

Comment:
Nit:
Typically those are printed in camel case i.e. `Json`
```suggestion
    fun getEthBalance() {
```